emails/2015/feb-meetup-call-for-speakers.md

```diff
@@ -0,0 +1,92 @@
+
+But all jokes aside: we're all relying on the community as we make our living using freely available tools, and free knowledge generously shared by previous speakers. Please seriously consider helping the community by giving a talk (and making yourself look cool and seem smart, which you are!).
+
+Of course, you can give a talk on any topic you want (it should be about Django, or at least something interesting for Django programmers). If you don't know what you want to give a talk on, there are many ideas below. If you don't know what to do, but feel the nudge to share your knowledge, respond to the email, and one of us will help you figure it out. If you don't want to give a talk, but are interested in something, let us know.
+
+Alright, don't even think about closing the inbox, don't defer it, just respond and say "yes! I'll give a talk." :-)
+
+Ideas on finding a talk:
+
+1. Fork a talk
+
+Many, many talks have been given in the past, so instead of starting from the ground up, you could "fork" somebody else's talk and save a bunch of time, while still bringing lots of benefit to us. With a simple email, it's likely that you could get the permission of the original author to use their slides, code, etc.
+
+Places to find talks:
+
+Djangocon 2014: https://www.youtube.com/watch?v=cqP758k1BaQ&list=PLE7tQUdRKcybbNiuhLcc3h6WzmZGVBMr3
+Djangocon 2013: https://www.youtube.com/watch?v=drWhGUdadGw&list=PLtqtTJ4wP09YOFqm_lBCoQtmS6S0omW3J
+Djangocon Europe 2014: http://eventifier.com/event/djangoconeuropw/videos?page=2
+Search SpeakerDeck for Django: https://speakerdeck.com/search?utf8=%E2%9C%93&q=django (this brings up 44 pages of talks!)
+
+2. Get an idea / content for a talk from a blog post
+
+The Django and Python world are full or really smart and generous people. Start with what they have written, and (ask first), and turn it into a talk. A good place to look is on the archives of popular newletters that aggregate Django and Python articles:
+
+The "Django Round Up" by Lincoln Loop is excellent: http://us2.campaign-archive1.com/home/?u=24fce4628ba1d5814debef334&id=2d6dd01daf
+Python Weekly archives: http://www.pythonweekly.com/archive/
+pycoder's archives: http://pycoders.com/archive/
+
+3. If that doesn't work, here's a list of ideas that we recently created as "potential interesting talks"
+
+- Django with Haystack/Elasticsearch
+- Django and Docker
+- Django and AngularJS
+- Flask vs Django
+- Getting a dev environment set up with Django + Vagrant + Virtualbox
+- Django for a (Rails / PHP / .NET / etc) programmer
+- High Performance Django (a talk that uses the recent book by Lincoln Loop to discuss making Django fast)
+- Testing in Django (Given many times. Never too many)
+- What's up with security in Django? (what do I need to be worried about? What does Django take care of for me)
+- Django Rest Framework
+- The state of real-time applications with Django (what are my options?)
+- Getting the most out of the Django ORM
+
```

Comment:
Do we want to link to a Google Form that posts to the existing Google Spreadsheet with talk topic ideas and potential speakers?